src/test/legacy/extension.test.ts

```diff
@@ -33,7 +33,8 @@ import {
 } from "../unit/stubs";
 import {Workspace} from "../../lib/workspace";
 
-suite('Extension Test Suite', () => {
+suite('Extension Test Suite', function () {
+    this.timeout(60000); // Global timeout for all tests in this suite
```

Comment:
I was seeing some more test failures so updated the global timeout to help with this